isolates/bin/long_running_isolate.dart

```diff
@@ -0,0 +1,88 @@
+// Spawn an isolate, read multiple files, send their contents to the spawned
+// isolate, and wait for the parsed JSON.
+import 'dart:async';
+import 'dart:convert';
+import 'dart:io';
+import 'dart:isolate';
+
+final files = [
+  File('json_01.json'),
+  File('json_02.json'),
+  File('json_03.json'),
+];
+
+// Reads multiple files and parses them on a separate isolate.
+Future<void> main() async {
```

Comment:
Might be nice to have @lrhn or @mkustermann review these!

Reply:
I'd make this `void main() async {`. There is no reason to suggest that anyone will ever see the result of calling `main`.